src/translations/en.json

```
@@ -1573,6 +1573,15 @@
           "system_health": {
             "manage": "Manage",
             "more_info": "more info"
+          },
+          "items": {
+            "change_log": "Change Log",
+            "thanks": "Thanks To",
+            "merch": "Merchandise",
+            "feature": "Feature Requests",
+            "bug": "Bug Report",
```

```suggestion
            "bug": "Bug Reports",
```

Since FRs are also plural above